app/assets/stylesheets/custom.scss

```diff
@@ -1 +1,5 @@
-@import "bootstrap/scss/bootstrap";
\ No newline at end of file
+@import "bootstrap/scss/bootstrap";
+
+body {
+  margin-top: 80px;
+}
```

Comment:
Convention là add một blank line vào end of file nhé
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file